ql/src/test/results/clientpositive/acid_table_directories_test.q.out

```diff
@@ -154,6 +154,7 @@ POSTHOOK: Input: default@acidparttbl@p=200
 ### ACID BASE DIR ###
 ### ACID BASE DIR ###
 ### ACID BASE DIR ###
+### ACID BASE DIR ###
```

Comment:
Is this expected?

Reply:
Enabling parallelism when inserting eliminates this change.